src/components/UI/Text/InputNumber.vue

```diff
@@ -53,7 +53,7 @@
         :class="[inputAreaClass, boxStyling]"
         :min="min"
         :max="max"
-        type="number"
+        type="type"
```

Comment:
Can you help me understand what is this supposed to do?
As far as I remember, There's no "type" type in an HTML `<input>` tag

Reply:
See this - https://www.w3schools.com/html/html_form_input_types.asp

Reply:
I hope the change answers the question